src/Compilers/Test/Utilities/VisualBasic/CompilationTestUtils.vb

```diff
@@ -12,9 +12,14 @@ Imports Microsoft.CodeAnalysis.Text
 Imports Roslyn.Test.Utilities
 Imports Roslyn.Test.Utilities.TestBase
 Imports Xunit
+Imports Microsoft.CodeAnalysis.Collections
 
 Friend Module CompilationUtils
 
+    Private ReadOnly _pooledStringBuilderPool As New ObjectPool(Of PooledStringBuilder)(
+        New ObjectPool(Of PooledStringBuilder).Factory(Function() PooledStringBuilder.GetInstance), 64)
```

Comment:
@AlekseyTs  I'll investigate, but it would be a lot simpler if there was an overload `.CreatePool(size as int32)`